### PR TITLE
[PRMP-1122] add patients to NRL SQS

### PR DESCRIPTION
**Pull request by jack-nhs**

### CODE REVIEW

File: lambdas/tests/unit/services/test_bulk_upload_service.py [hunk 1]
```diff
@@ -190,6 +190,7 @@ def test_lambda_handler_handle_pds_too_many_requests_exception(
         mock_back_to_queue.assert_any_call(message)
 
 
+@pytest.mark.focus
```
Comment (AndyFlintNHS) [inline lambdas/tests/unit/services/test_bulk_upload_service.py:193]: What is this? I've taken a look at this on google and it looks like it forces pytest to run this test exclusively, if I've understood correctly, not something we'd want to commit

File: lambdas/repositories/bulk_upload/bulk_upload_sqs_repository.py [hunk 2]
```diff
@@ -39,3 +40,6 @@ def put_sqs_message_back_to_queue(self, sqs_message: dict):
             message_body=sqs_message["body"],
             nhs_number=nhs_number,
         )
+
+    def send_message_to_nrl_fifo(self, queue_url: str, message: NrlSqsMessage):
+        self.sqs_repository.send_message_standard(queue_url, message.model_dump())
```
Comment (NogaNHS) [inline lambdas/repositories/bulk_upload/bulk_upload_sqs_repository.py:45]: This is using `send_message_standard` not the fifo queue

Comment (NogaNHS) [inline lambdas/repositories/bulk_upload/bulk_upload_sqs_repository.py:45]: need to add a unit test

File: lambdas/services/bulk_upload_service.py [hunk 3]
```diff
@@ -269,6 +272,16 @@ def handle_sqs_message(self, message: dict):
...
         )
 
+        NrlMessage = NrlSqsMessage(
+            nhs_number=staging_metadata.nhs_number, action=NrlActionTypes.CREATE
+        )
+        self.sqs_repository.send_message_to_nrl_fifo(
+            queue_url=self.nrl_queue_url,
+            message=NrlMessage,
+            nhs_number=staging_metadata.nhs_number,
+            group_id=f"nrl_sqs_{uuid.uuid4()}",
```
Comment (NogaNHS) [inline lambdas/services/bulk_upload_service.py:282]: this need testing

**Review by AndyFlintNHS — DISMISSED** [review]

**Review by AndyFlintNHS — APPROVED** [review]

**Review by oliverbeumkes-nhs — APPROVED** [review]

**Pull request merged**